src/OrbitGl/GraphTrack.h

```diff
@@ -41,6 +41,12 @@ class GraphTrack : public Track {
   void SetLabelUnitWhenEmpty(const std::string& label_unit);
   void SetValueDecimalDigitsWhenEmpty(uint8_t value_decimal_digits);
 
+  void OnTimer(const orbit_client_protos::TimerInfo& timer_info) override;
+  std::vector<std::shared_ptr<TimerChain>> GetAllChains() const override;
+  std::vector<std::shared_ptr<TimerChain>> GetAllSerializableChains() const override {
```

Comment:
Since now all `Track` subclasses implement these 2 methods, you could think about making them pure virtual in `Track.h`

Reply:
Don't forget the lovely FrameTrack!

Reply:
Thanks for the suggestions! Changed accordingly.